docs/sources/insight-logs/_index.md

```diff
@@ -0,0 +1,101 @@
+# Grafana OnCall insight logs
+
+Grafana OnCall insights logs represents certain activities, such as:
+
+- A user creates, updates or deletes resource.
+- A Maintenance mode is started or finished for an integration.
+- A user configure ChatOps integration.
```

Comment:
I don't like this description, it is too wide.  Actually chatops insight logs created every time when user link/unlink messaging backend, link/unlink channel or install slack, but it seems to long to me